### ISSUE TITLE
1px border when window maximize?

### ISSUE THREAD

**Issue by kmvan** [fakedeltatime/flat-two-color#1]
Click the top right corner can not close window. It's a 1px border.
----
Xfce4.14 Greybird

**Comment by fakedeltatime** [comment]
There seems to be one pixel to the right of the close button? I could increase the clickable area of the button so that the one pixel is included, although I think I'd prefer having a wider gap while maximized to match the look when it isn't maximized. Is the problem that it seems like you should be able to click it, or is it preferrable to be able to click it at the edge of the window while maximized?

**Comment by kmvan** [comment]
When the window is maximized, is it expected to click the close button in the top right corner?

**Comment by fakedeltatime** [comment]
Yes, to close the window, if the close button is positioned there. You didn't provide additional information so I'm not sure about the problem anymore.

**Comment by kmvan** [comment]
I upload a screen record, maybe helpful.
https://www.youtube.com/watch?v=RFYJ5kTa33c&feature=youtu.be

**Comment by fakedeltatime** [comment]
Well to me that's what should happen based on the visual feedback and the size of the box, and it's also what I'm used to with the themes I usually use and what I based this theme off of.
